.github/workflows/ci.yml

```diff
@@ -125,7 +125,7 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: test
-          args: --no-fail-fast
+          args: --no-fail-fast --all-targets
```

Comment:
this, interestingly, causes doctests not to run

Reply:
interesting, without it, it causes the tests in examples to not run.

Reply:
https://github.com/rust-lang/cargo/issues/6669

Reply:
added two steps, one with --all-targets and one with --doc